.npmrc

```diff
@@ -1 +1,4 @@
-package-lock=false
\ No newline at end of file
+//registry.npmjs.org/:_authToken=${NODE_AUTH_TOKEN}
+registry=https://registry.npmjs.org/
+always-auth=true
```

Comment:
I think this will need to be removed when other people start working on the project, but that can be tackled at a different time.

Reply:
Sounds good, let's revisit this in the future